build_tools/github_actions/build_dist.py

```diff
@@ -238,6 +245,7 @@ def bazel_build_tf_binary(target):
 def build_py_xla_compiler_tools_pkg():
   """Builds the iree-install/python_packages/iree_tools_xla package."""
   install_python_requirements()
+  configure_bazel()
```

Comment:
We should either have this in `bazel_build_tf_binary` or in each place that calls it. Not both